packages/main/src/models/Address.ts

```diff
@@ -44,15 +44,17 @@ export function build({
 }: {
   email: string
   name?: string
-}): Address {
-  const [mailbox, host] = email.split("@", 2)
-  return name
-    ? new Address({
-        name,
-        mailbox,
-        host
-      })
-    : new Address({ mailbox, host })
+}): Address | null {
+  const parsed = addrs.parseOneAddress(email)
```

Comment:
My solution to the bug of invalid email addresses is to use a proper email parsing library instead of the quick solution I had previously. `parseOneAddress` will return `null` if its input is not an email address.